README.md

```diff
@@ -103,10 +108,17 @@ A curated list of awesome TinyGo projects. Inspired by [awesome-go](https://gith
 
 * [TinyWebSocket](https://github.com/Nerzal/tinywebsocket) - Wraps the `syscall/js` api to provide a better developer experience, while being compilable by TinyGo
 
+### WASI and WASM Unknown
+
+* [go-modules](https://github.com/bytecodealliance/go-modules) - WebAssembly, WASI, and Component Model tools for Go
+* [jsony](https://github.com/orsinium-labs/jsony) - A blazing fast and safe Go package for serializing JSON
```

Comment:
jsony isn't wasm specific... should it go somewhere else, e.g. Awesome Library or protocol sections, or maybe a new Encoding section?  Not sure.